react/lib/components/Widget/WidgetContainer.tsx

```diff
@@ -108,6 +105,9 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
       contributionOffset,
       ...widgetProps
     } = props;
+    const [internalCurrencyObj, _setInternalCurrencyObj] = useState<CurrencyObject>();
```

Comment:
Setter shouldn't have been prefixed with a underscore, it is used in the line below